Service Accelerator Template for PostgreSQL on Microsoft Azure (#60)
finos/cloud-service-certification: azure/postgresql/ServiceApprovalAcceleratorPostgreSQL.md

            <li>SOC1</li>
            <li>SOC2</li>
            <li>SOC3</li>
            <li>EU Model Clauses</li>   
            <li>UK G-cloud</li>
        </ul>
    </td>
    <td class="tg-0lax"><a href="https://azure.microsoft.com/fr-fr/blog/compliance-offerings-for-azure-database-for-mysql-and-azure-database-for-postgresql/">Article on Compliance</a></td>
  </tr>
  <tr>
    <td class="tg-7btt" rowspan="3">Identity &amp; Access Management</td>
    <td class="tg-fymr">Authentication</td>
    <td class="tg-f8tv">
        <p>
            It is recommended to enable AAD integration with PAzure Database for PostgreSQL.
            <br><br>
            Microsoft Azure Active Directory (Azure AD) authentication is a mechanism of connecting to Azure Database for PostgreSQL using identities defined in Azure AD.
            With Azure AD authentication, you can manage database user identities and other Microsoft services in a central location, which simplifies permission management.            
            <br><br>
            when using Azure AD for authentication, the security features that build in AAD can help to protect the login process. The AAD token can be obtained in various

>             ways which could help to protect most of the password attach. 

Reviewer: Is there a typo here? "Protect most of the password attach"

Reviewer 2: +1

            <br><br>
            Azure managed identity can be used by many azure resources (VM, app services, azure function, etc) to have a controled identity within AAD, which elimate the needs store and
            manage any form of password or token for deployed application. It further strenghtens the protection for authentication.
            <br><br>
            When AAD integration is not enabled, simple user/pass is used for authentication in postgres.
         </p>
    </td>
    <td class="tg-0pky"><a href="https://docs.microsoft.com/en-us/azure/postgresql/concepts-aad-authentication">Use Azure Active Directory for authenticating with PostgreSQL</a></td>
  </tr>
  <tr>
    <td class="tg-fymr">Authorization</td>
    <td class="tg-f8tv">
        <p>

>             It is recommended to only assign AAD groups roles within postgres, and not to create indivisual users within postgres itself.            

Reviewer: ```suggestion
            It is recommended to only assign AAD groups roles within postgres, and not to create individual users within postgres itself.            
```

            <br><br>
            When setting up AAD integration for Azure postgres, an AAD account (either group or user) needs to be specified as the azure ad administorator for the postgres server.             

>             Only the AAD users that are specified can initially connect and add more AAD user/group into postgres and assign database roles.            

Reviewer: ```suggestion
            Only the AAD users that are specified can initially connect, add more AAD users/groups into postgres and assign database roles.            
```

Reviewer 2: +1

            <br><br>
            Custom database roles with least privileged access is always preferable, logically grouped permissions can be used to ease the burdon of administration
            <br><br>                        

>             When a AAD group is added into the postgres server, any member of the group can login and assume the database role that is assigned to the group. It thus recommended to create and add 

Reviewer: ```suggestion
            When a AAD group is added into the postgres server, any member of the group can login and assume the database role that is assigned to the group. It is therefore recommended to create and add 
```

>             AAD groups only to postgres, and manage all users and role assignments using AAD, and eleminate the needs to create individual users within postgres.            

Reviewer: ```suggestion
            AAD groups only to postgres, and manage all users and role assignments using AAD. This eleminates the needs to create individual users within postgres.            
```

        </p>
    </td>
    <td class="tg-0pky"></td>
  </tr>
  <tr>
    <td class="tg-fymr">Privileged Access Management</td>
    <td class="tg-f8tv">
        <p>
            Only authorised Database Administrators group members should be allowed to create or update or delete any instance of Azure database for PostgreSQL.

>             Additionally, this should happen in accordance to the defined deployment process as well (e.g. via a build service, etc.).

Reviewer: ```suggestion
            Additionally, this should happen in accordance with the defined deployment process (e.g. via a build service, etc.).
```

            <li>CSA STAR Attestation</li>
            <li>CSA STAR Certification</li>
            <li>HIPAA / HITECH Act</li>
            <li>PCI DSS Level 1</li>
            <li>SOC1</li>
            <li>SOC2</li>
            <li>SOC3</li>
            <li>EU Model Clauses</li>   
            <li>UK G-cloud</li>
        </ul>
    </td>
    <td class="tg-0lax"><a href="https://azure.microsoft.com/fr-fr/blog/compliance-offerings-for-azure-database-for-mysql-and-azure-database-for-postgresql/">Article on Compliance</a></td>
  </tr>
  <tr>
    <td class="tg-7btt" rowspan="3">Identity &amp; Access Management</td>
    <td class="tg-fymr">Authentication</td>
    <td class="tg-f8tv">
        <p>
            It is recommended to enable AAD integration with PAzure Database for PostgreSQL.
            <br><br>
            Microsoft Azure Active Directory (Azure AD) authentication is a mechanism of connecting to Azure Database for PostgreSQL using identities defined in Azure AD.
            With Azure AD authentication, you can manage database user identities and other Microsoft services in a central location, which simplifies permission management.            
            <br><br>

>             when using Azure AD for authentication, the security features that build in AAD can help to protect the login process. The AAD token can be obtained in various

Reviewer 2: "When"

            Connections to an Azure Database for PostgreSQL server are first routed through a regional gateway. The gateway has a publicly accessible IP, while the server
            IP addresses are protected. The following diagram illustrates the traffic flow:</p>
        <img src="media/net-security.png" width="700" height="300">
        <p>
            A newly created Azure Database for PostgreSQL server has a firewall that blocks all external connections. Though they reach the gateway, they are not allowed to
            connect to the server. <br><br>
            To begin using your server from another computer, you need to specify one or more server-level firewall rules to enable access to your server. Use the firewall rules
            to specify which IP address ranges from the Internet to allow. Access to the Azure portal website itself is not impacted by the firewall rules. <br><br>
            The example uses Access to Azure Services enabled or Public peering. This will be different for private link but gives some insight to how communications are proxied
            to Azure PAAS services.<br><br>
        </p>
        <h4>Connecting from Azure</h4>
        <p>

>             To allow applications from Azure to connect to your Azure Database for PostgreSQL server, Azure connections must be enabled. For example, to host an Azure Web Apps

Reviewer 3: This is written strangely. It starts by saying that you need to enable azure connections to achieve (connecting from other apps in Azure). But the end of the paragraph is actually recommending to disable azure connections (and the subsequent point on data prevention also says best practice is to disable).  The consistency of what is being recommended isn't clear.
